persistence/sql/sql_process_store.go

```diff
@@ -658,7 +662,13 @@ func (p sqlProcessStoreImpl) doCompleteWaitUntilExecutionTx(
 		LastFailure:        nil,
 	}
 
-	err := tx.UpdateAsyncStateExecution(ctx, stateRow)
+	// lock process execution row first
```

Comment:
Actually, maybe just not lock it for now. Because this method is not updating the process execution at all. 
Can you just leave a comment here for the future: if this method is going to update process execution, it should lock it before updating or locking the async state executions. 

We will probably need to do that very soon, when @zklgame  is adding the localQueue command. But we may not need to update processExe for timerCommand, or no command.

Reply:
yep, I noticed this too, but waitUntil will need to do it

Reply:
should we just leave it there?